pkg/option/config.go

```diff
@@ -2494,6 +2500,38 @@ func (c *DaemonConfig) populateDevices() {
 	}
 }
 
+func (c *DaemonConfig) populateLoadBalancerSettings() {
```

Comment:
I think `BindEnvWithLegacyEnvFallback()` achieves the same logic but in a simpler way (albeit with silent preference for the newer option)?

At a glance, the logic below looks right.